kernel/comps/logger/src/default_logger.rs

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use log::{Metadata, Record};
+use ostd::timer::Jiffies;
+use spin::Once;
+
+/// Default logger of this crate.
```

Comment:
I don't quite understand why this logger is the "default" logger. Does it mean that you intend to add another logger in the Asterinas kernel to override this logger? If so, why not modify the logger directly, since it is already a logger in the Asterinas kernel? If not, the logger is not a "default" logger, it's just the final implementation of the logger.

Reply:
I did this with the consideration that some logger functionalities might introduce certain performance overhead, such as the simple implementation of a filter logger. This way, users can control the injection of different functional loggers directly through features. 

However, now I realize that it could also be directly modified within the logger's implementation, with feature switches to enable or disable corresponding functionalities. This approach could also allow for the combination of different features.